nemo_skills/inference/generate.py

```diff
@@ -525,6 +529,20 @@ def dump_outputs(self, outputs, data_points, fout):
         for output in outputs:
             fout.write(json.dumps(output) + "\n")
 
+    def drop_binary_data(self, output):
+        """Remove binary data (like base64 audio) from messages to keep output files smaller."""
+        # Skip if output doesn't have messages (e.g., text completion mode or error cases)
+        if "messages" not in output:
+            return
+
+        for message in output["messages"]:
+            # Skip if content is not a list (e.g., string content in system messages)
+            if not isinstance(message.get("content"), list):
+                continue
+
+            # Filter out audio_url items from list-style content
+            message["content"] = [content for content in message["content"] if content.get("type") != "audio_url"]
```

Comment:
can you make "audio_url" here a list that is specified in this config? that way it can be configurable and other audio-related fields can be included/excluded if desired